runtime/queries/rust/highlights.scm

```diff
@@ -40,9 +40,10 @@
 (enum_variant (identifier) @type.enum.variant)
 
 (field_initializer
-  (field_identifier) @property)
-(shorthand_field_initializer) @variable
-(shorthand_field_identifier) @variable
+  (field_identifier) @field)
+(shorthand_field_initializer
+  (identifier) @field)
+; (shorthand_field_identifier) @field
```

Comment:
I'm fairly certain that `shorthand_field_identifier` is a legitimate node, but I don't recall where I encountered it. However, `shorthand_field_initializer` should probably still be considered a variable because it's initialized with a variable.